### ISSUE TITLE
Must specify a valid bucketing strategy while requesting aggregation

### ISSUE THREAD

**Issue by nielstj** [krokyze/FitKit#7]
Thanks for writing this plugin.

I encounter Platform exception when trying to fetch steps data with this code :

```
final start = startOfDay(date);
final end = endOfDay(date);
final data = await FitKit.read(DataType.STEP_COUNT, start, end);
```

The exception

```
 PlatformException(FitKit, Must specify a valid bucketing strategy while requesting aggregation, null)
```

Tried solution from [here](https://stackoverflow.com/questions/38608174/java-lang-illegalstateexception-must-specify-a-valid-bucketing-strategy-while-r), by removing bucketByTime(...),  no exception thrown, but results always empty or 0.

Is this a problem with the data aggregation? Thanks in advance.

**Comment by krokyze** [comment]
Hey @nielstj 

Sorry for the late reply. Did you already manage to solve this issue? What's the start/end dates?

**Comment by nielstj** [comment]
No, I haven't resolve it yet. Start is the beginning of a day at 00.00 and end is the end of day at 23.59.
